### PR TITLE
(chore) rename Tint => Tint Terminal

### PR DESCRIPTION
**Pull request by joshgoebel**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package _adds only bindings for scratch views it creates._
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view. _The upcoming update resolves this._
- [x] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

This is just the renaming (and refreshing of a package already in the library).

[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

File: repository/t.json [hunk 1]
```diff
@@ -2375,19 +2375,23 @@
 			]
 		},
 		{
-			"name": "Tint",
-			"details": "https://github.com/yyyc514/Tint",
-			"author": "jgoebel",
+			"name": "Tint Terminal",
+			"previous_names": ["Tint"],
+			"details": "https://github.com/joshgoebel/sublime-tint-terminal",
+			"author": "joshgoebel",
```
Comment (braver) [inline repository/t.json:2381]: Since the author here is just your GitHub username, you can leave it out.

Comment (joshgoebel) [reply repository/t.json:2381]: I see a lot of people using their liternal name perhaps I should change it to "Josh Goebel" then?

Comment (braver) [reply repository/t.json:2381]: Yes, you can also do that 👍🏻

Comment (joshgoebel) [reply repository/t.json]: Better?

Comment (braver) [reply repository/t.json]: 👍🏻

**Pull request merged**